src/net_processing.cpp

```diff
@@ -5818,6 +5828,29 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                     // No reason to drain out at many times the network's capacity,
                     // especially since we have many peers and some will draw much shorter delays.
                     unsigned int nRelayedTransactions = 0;
+
+                    size_t inbounds_nonrcncl_tx_relay = 0, outbounds_nonrcncl_tx_relay = 0;
```

Comment:
for code readability, such variables could be renamed `inbounds_flooding_tx_relay` / `outbounds_flooding_tx_relay`, which is matching the low-fanout flooding strategy used for them.